examples/qec_decoder_toolkit/Quantinuum_hseries_qec_decoder_toolkit.ipynb

```diff
@@ -155,11 +176,11 @@
    "source": [
     "### 2. Define classical functions in your language of choice\n",
     "\n",
-    "Define the classical functions you plan to use in your hybrid compute program. These will be the classical functions that your quantum program will run during execution. In Rust, edit the `src/lib.rs` file. For C/C++, edit the `src/lib.c` file. \n",
+    "Define the classical functions you plan to use in your hybrid compute program. These will be the classical functions that your quantum program will run during execution. In Rust, edit the src/lib.rs file. For C/ C++, edit the src/lib.c file. \n",
     "\n",
     "Examples of the types of functions that can be put in Rust or C code are provided in the respective folders within each of the example folders. *We recommend looking at one of the provided examples to know which functions and attributes are required for Hybrid Compute to work.*\n",
     "\n",
-    "*Note:* The standard C (C++) library cannot be used in a program that is intended to be compiled to Wasm."
+    "*Note:* The standard C/ C++ library cannot be used in a program that is intended to be compiled to Wasm."
```

Comment:
Have you discussed this with Selwyn?

Reply:
I'm aware of Selwyn's response. I check with Hardware and there was no response.